pom.xml

```diff
@@ -637,12 +637,12 @@
 
   <issueManagement>
     <system>Trac</system>
-    <url>http://trac.openmicroscopy.org.uk/ome/wiki/BioFormats</url>
+    <url>https://trac.openmicroscopy.org/ome/wiki/BioFormats</url>
```

Comment:
Or maybe a link to http://www.openmicroscopy.org/site/support/bio-formats/about/bug-reporting.html ?